sdks/python/apache_beam/io/filesystem.py

```diff
@@ -295,6 +295,9 @@ def read(self, num_bytes: Optional[int] = None) -> bytes:
     if not self._decompressor:
       raise ValueError('decompressor not initialized')
 
+    if num_bytes is None:
+      num_bytes = DEFAULT_READ_BUFFER_SIZE
```

Comment:
Thanks! Rather than setting this here, can we just set the default value in the function definition?